### ISSUE TITLE
High contrast mode

### ISSUE THREAD

**Issue by CharlesAverill** [CharlesAverill/satyrn#46]
Make a high contrast mode in addition to light mode #37

**Comment by alexdanowicz** [comment]
CSS contrast will do the work, let me help.

**Comment by CharlesAverill** [comment]
@Danowicz Absolutely! Could you add a sub-dropdown in the View dropdown for Themes, and submit a PR when you're ready? Thank you!
